Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/heapster",
-	"GoVersion": "go1.6",
+	"GoVersion": "go1.7",
```

Comment:
This doesn't actually change the compiler, IIRC -- it's basically just a record of which version of go was used to run Godep.  Please set it back.